src/einsteinpy/symbolic/helpers.py

```diff
@@ -46,6 +46,61 @@ def simplify_sympy_array(arr):
         return sympy.Array(sympy.simplify(sum(arr)))
 
 
+def discard_terms_sympy_array(arr, discard_terms):
```

Comment:
If this function "adapts / approximates" arrays, can this function be called `adapt_sympy_array` or `approx_sympy_array`, unless those terms seem to conflate the actual purpose? Just to keep the naming in line with other functions here - `simplify_*` & `expand_*`.

Reply:
I actually recently found the applyfunc() of sympy arrays, that is exactly what I wanted to do so I removed the function in question